common/app/staticpages/StaticPages.scala

```diff
@@ -19,7 +19,7 @@ object StaticPages {
       section = Option(SectionSummary(id="email-signup-page", activeBrandings=None)),
       webTitle = webTitle,
       analyticsName = "email-signup-page",
-      contentType = "signup",
+      contentType = "Signup",
```

Comment:
In general stuff like this might be better done with a Sum Type (sealed trait extended by case objects) rather than an arbitrary string. Then you get type safety since the compiler can tell you what the possible values of `contentType` are. Case issues such as these become irrelevant because if you use the wrong case, the code won't compile in the first place. (There may be a valid reasons why that wouldn't work in this case though)